### PR TITLE
Add quickstart and install to readme

### LINKED ISSUE #144: Short example in the readme
I think it would be helpful to have a very bare-bones example in the readme using porter.yaml, porter create, porter build, duffle build, etc.

This would help to explain to a newcomer who lands on this repo exactly what they might use Porter to accomplish.

### PR DESCRIPTION
**Pull request by jdolitsky**
Closes #144.

Was trying out Porter+Duffle, added a few things to the readme that would have helped me to know in advance.

I'm not 100% sure about the accuracy of the "Running your installer using Duffle" section I added, maybe you can clarify. Still not fully clear how the two tools are designed to play together.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,118 @@
...
+```
+
+The `duffle list` command can be used to show all installed applications.
+
+If you wish to uninstall the application, you can use `duffle uninstall`:
+```
+duffle uninstall demo
+```
+
+## Installation
```
Comment (carolynvs) [inline README.md]: This is already documented on the website at https://porter.sh/install/.

File: README.md [hunk 2]
```diff
@@ -3,14 +3,118 @@
...
 focus on what you know best: your application.
 
 Learn more at [porter.sh](https://porter.sh)
 
----
+*Want to work on Porter with us? See our [Contributing Guide](CONTRIBUTING.md)*
 
-Want to work on Porter with us? See our [Contributing Guide](CONTRIBUTING.md)
\ No newline at end of file
+## Getting Started
```
Comment (carolynvs) [inline README.md]: Just a heads up that the porter docs all live at https://porter.sh/docs/ which is why the readme points people there. We are slowly building up docs over there, and some already exist (such as the install guide).

Would you please move the getting started to a new file located at /docs/content/quickstart.md and then link to it from the README here?

The readme can have links to useful stuff for new contributors, but I'd like to avoid repeating content so that we aren't dual editing. 😀 

Thank you for adding this! 🎉

File: README.md [hunk 3]
```diff
@@ -3,14 +3,118 @@
...
+# Porter is a tool for building cloud installers
 
 Porter gives you building blocks to create a cloud installer for your application, handling all the
 necessary infrastructure and configuration setup. It is a declarative authoring experience that lets you
 focus on what you know best: your application.
 
 Learn more at [porter.sh](https://porter.sh)
 
----
+*Want to work on Porter with us? See our [Contributing Guide](CONTRIBUTING.md)*
```
Comment (carolynvs) [inline README.md]: 🙇‍♀️

File: README.md [hunk 4]
```diff
@@ -3,14 +3,118 @@
 [![Build Status](https://dev.azure.com/cnlabs/porter/_apis/build/status/deislabs.porter?branchName=master)](https://dev.azure.com/cnlabs/porter/_build/latest?definitionId=6?branchName=master)
 
 
-# Porter is a Cloud Installer
+# Porter is a tool for building cloud installers
```
Comment (carolynvs) [inline README.md]: Can you revert this change for now? I don't want to change how we are branding/explaining Porter in this PR.

**Comment by jdolitsky** [comment]
@carolynvs thanks for review, comments addressed!

File: README.md [hunk 5]
```diff
@@ -9,8 +9,10 @@ Porter gives you building blocks to create a cloud installer for your applicatio
 necessary infrastructure and configuration setup. It is a declarative authoring experience that lets you
 focus on what you know best: your application.
 
+Want to start using Porter? Check out the [Quickstart Guide](docs/content/quickstart.md) for a brief walkthrough.
```
Comment (carolynvs) [inline README.md]: The link should go to https://porter.sh/quickstart/

You can preview your doc changes at https://deploy-preview-157--porter.netlify.com/quickstart/ (I got the link from the netlify preview check at the bottom of the PR).

File: docs/content/quickstart.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
+---
+title: Quickstart Guide
+descriptions: Get started using Porter
+---
+
+## Getting Porter
+
+First make sure Porter is installed.
+Please see the [installation instructions](./install.md) for more info.
```
Comment (carolynvs) [inline docs/content/quickstart.md]: The link should go to `/install/` which will take you to https://porter.sh/install/. You can [preview the doc site locally with Hugo](https://github.com/deislabs/porter/blob/master/CONTRIBUTING.md#documentation) or just rely on the netlify preview.

The reason why I'm asking for the change is that linking to the md files directly (as we are used to for linking within GH) is that Hugo doesn't handle those links properly, and it results in a 404 on the real doc site.

File: docs/content/quickstart.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+## Creating a new installer
```
Comment (carolynvs) [inline docs/content/quickstart.md]: This should be `Create a new bundle`.

We haven't made a decision to not use the same term as [CNAB](https://cnab.io) for bundles. So while informally on the landing page we called it a cloud installer, we aren't ready to change the terms everywhere. It would be confusing I think. @jeremyrickard ?

File: docs/content/quickstart.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+uninstall:
+  - description: "Uninstall Hello World"
+    exec:
+      command: bash
+      arguments:
+        - -c
+        - echo Goodbye World
+```
+
+## Building a CNAB bundle
```
Comment (carolynvs) [inline docs/content/quickstart.md]: nit: Let's change this to "Build the Bundle". I try to not reference CNAB for the most part unless it is necessary.

File: docs/content/quickstart.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+
+The `porter build` command will create a [CNAB-compliant](https://github.com/deislabs/cnab-spec/blob/master/101-bundle-json.md) `bundle.json`, as well as build and push the associated invocation image:
+```
+porter build
+```
+
+Note: Make sure that the `invocationImage` listed in you `porter.yaml`  is a reference that you are able to `docker push` to and that your end-users are able to `docker pull` from.
```
Comment (carolynvs) [inline docs/content/quickstart.md]: 👍 Thanks for calling this out!

File: docs/content/quickstart.md [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+
+## Running your installer using Duffle
```
Comment (carolynvs) [inline docs/content/quickstart.md]: Maybe change this to "Install the bundle"?

File: docs/content/quickstart.md [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+
+_Wondering the differences between Duffle and Porter? Please see [this page](./porter-or-duffle.md)._
```
Comment (carolynvs) [inline docs/content/quickstart.md]: Let's remove lines 57 through 65, basically skipping to the duffle install part. 

We are working on wrapping duffle install into porter, but we aren't there yet. 😀

Comment (carolynvs) [reply docs/content/quickstart.md]: I'll update the porter-or-duffle file with our new direction. In summary: porter builds on top of duffle, and uses its libraries. For now we use porter to build the bundle, and duffle to install it.

duffle.json isn't necessary which is why I asked you to skip the rebuilding with duffle. The porter build command creates a CNAB compliant bundle already, which duffle can install without further tweaks.

File: docs/content/quickstart.md [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+
+The file `duffle.json` is required by Duffle. Since both Porter and Duffle are CNAB-complaint, we can simply reuse the `bundle.json` created during `porter build` and run `duffle build` to save it to the local store:
+```
+cp bundle.json duffle.json
+duffle build .
+```
+
+You can view all bundles in the local store with `duffle bundle list`.
+
+Afterwords, use `duffle install` to run your installer ("demo" is the unique installation name):
```
Comment (carolynvs) [inline docs/content/quickstart.md]: We should include telling people how to install duffle: https://github.com/deislabs/duffle/blob/master/README.md#getting-started

**Comment by jdolitsky** [comment]
ready for re-review. thanks!

**Comment by carolynvs** [comment]
Looks great, thank you! 🍰
